packages/apollo-federation/src/composition/compose.ts

```diff
@@ -315,6 +306,7 @@ export function addFederationMetadataToSchemaNodes({
     { serviceName: baseServiceName, extensionFieldsToOwningServiceMap },
   ] of Object.entries(typeToServiceMap)) {
     const namedType = schema.getType(typeName) as GraphQLNamedType;
+    if (!namedType) continue;
```

Comment:
Why was this added? (can we add a comment?)

Reply:
todo: fix typecasting/comment/typeToServiceMap